python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -112,6 +112,11 @@ def start_ganga(gangadir_for_test, extra_opts=[]):
     if missing_cred:
         raise Exception("Failed due to missing credentials %s" % str(missing_cred))
 
+    # Make sure that all the config options are really set.
+    # Some from plugins may not have taken during startup
```

Comment:
The issue is that if a test sets `[DIRAC]DiracEnvFile='something'` in its `setUp` then this function will pass it to `setConfigOption`. However, if at that point the DIRAC plugin has not been loaded, this will fail silently and when the test starts, the setting will not have been set. By line 115 the plugin will have been loaded and so it is safe to set it to the requested value.